content/chainguard/chainguard-images/getting-started/wordpress.md

```diff
@@ -0,0 +1,305 @@
+---
+title: "Getting Started with the WordPress Chainguard Image"
+type: "article"
+linktitle: "WordPress"
+description: "Tutorial on how to get started with the WordPress Chainguard Image"
+date: 2024-07-19T11:07:52+02:00
+lastmod: 2024-07-19T11:07:52+02:00
+tags: ["Chainguard Images", "Products"]
+draft: false
+images: []
+menu:
+  docs:
+    parent: "getting-started"
+weight: 610
+toc: true
+---
+
+The [WordPress](https://images.chainguard.dev/directory/image/wordpress/overview) Chainguard Image is a container image suitable for creating and running WordPress projects. Designed to work as a drop-in replacement for the official [WordPress FPM-Alpine image](https://hub.docker.com/_/wordpress), the Chainguard WordPress image features a [distroless](/chainguard/chainguard-images/getting-started-distroless/) variant for increased security on production environments. The image is built with the latest PHP and WordPress versions, and includes the necessary PHP extensions to run WordPress.
+
+In this guide, we'll demonstrate 3 different ways in which you can use the WordPress Chainguard Image to build and run WordPress projects.
+
+## Preparation
+This tutorial requires Docker to be installed on your local machine. If you don't have Docker installed, you can download and install it from the [official Docker website](https://docs.docker.com/get-docker/).
+
+### Cloning the Demos Repository
+Start by cloning the demos repository to your local machine:
+
+```shell
+git clone git@github.com:chainguard-dev/edu-images-demos.git
```

Comment:
That's a good point! I don't think we have a convention, but as a general best practice I would think using SSH would be the way to go. We can bring this up at the weekly, for now I think I will keep SSH just in case.